PSO2Launcher/MainWindow.xaml

```diff
@@ -511,53 +515,5 @@
                 </Style>
             </Rectangle.Style>
         </Rectangle>
-        <Rectangle x:Name="Twitter" Height="40" Width="40" Margin="155,449,157,20" MouseDown="Twitter_MouseDown" ToolTip="TeamDogstar Twitter Account">
-            <Rectangle.OpacityMask>
-                <VisualBrush Stretch="Uniform"
-        			Visual="{StaticResource appbar_twitter_bird}"/>
-            </Rectangle.OpacityMask>
-            <Rectangle.Style>
-                <Style TargetType="{x:Type Rectangle}">
-                    <Setter Property="Fill" Value="{DynamicResource AccentColorBrush2}"/>
-                    <Style.Triggers>
-                        <Trigger Property="IsMouseOver" Value="True">
-                            <Setter Property="Fill" Value="{DynamicResource AccentColorBrush}"/>
-                        </Trigger>
-                    </Style.Triggers>
-                </Style>
-            </Rectangle.Style>
-        </Rectangle>
-        <Rectangle x:Name="Donate" Height="40" Width="40" Margin="202,447,110,22" MouseDown="Donate_MouseDown" ToolTip="Donate">
-            <Rectangle.OpacityMask>
-                <VisualBrush Stretch="Uniform"
-        			Visual="{StaticResource appbar_money}"/>
-            </Rectangle.OpacityMask>
-            <Rectangle.Style>
-                <Style TargetType="{x:Type Rectangle}">
-                    <Setter Property="Fill" Value="{DynamicResource AccentColorBrush2}"/>
-                    <Style.Triggers>
-                        <Trigger Property="IsMouseOver" Value="True">
-                            <Setter Property="Fill" Value="{DynamicResource AccentColorBrush}"/>
-                        </Trigger>
-                    </Style.Triggers>
-                </Style>
-            </Rectangle.Style>
-        </Rectangle>
-        <Rectangle x:Name="Information" Height="40" Width="40" Margin="249,445,63,24" MouseDown="Information_MouseDown" ToolTip="Get Information On IRC">
-            <Rectangle.OpacityMask>
-                <VisualBrush Stretch="Uniform"
-        			Visual="{StaticResource appbar_information_circle}"/>
-            </Rectangle.OpacityMask>
-            <Rectangle.Style>
-                <Style TargetType="{x:Type Rectangle}">
-                    <Setter Property="Fill" Value="{DynamicResource AccentColorBrush2}"/>
-                    <Style.Triggers>
-                        <Trigger Property="IsMouseOver" Value="True">
-                            <Setter Property="Fill" Value="{DynamicResource AccentColorBrush}"/>
-                        </Trigger>
-                    </Style.Triggers>
-                </Style>
-            </Rectangle.Style>
-        </Rectangle>
     </Grid>
 </controls:MetroWindow>
```

Comment:
We had to revert changes to this file to fix the progress bar. Does this button removal also fix the progress bars?